code-challenges.js

```diff
@@ -9,31 +9,95 @@
 
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 42
-// var temp = 350
-// var temp = 212
+var temp1 = 42
+var temp2 = 350
+var temp3 = 212
+
+var temp = [temp1, temp2, temp3]//Created an array for the three variables for temperature
+
+
+//Create a iteration of the array and output a statement based on the temp[index] value.
+for (let i = 0; i < temp.length; i++) {
+
+    if (temp[i] > 212) {
+        console.log(`${temp[i]} is above boiling point`)
+    }
+    else if (temp[i] < 212) {
+        console.log(`${temp[i]} is below boiling point`)
+    }
+    else { console.log(`${temp[i]} is at boiling point`) }
+}
+
+
```

Comment:
You definitely have the concepts down. But the goal of the challenges is to use the exact input to create the output.